### ISSUE TITLE
Mentions of Foundation for Public Code to possibly address

### ISSUE THREAD

**Issue by Ainali** [standard-for-public-code/standard-for-public-code#1024]
As this repository has been transferred from the [Foundation for Public Code namespace](https://github.com/publiccodenet/), there might be a couple of places where the Foundation is mentioned that this community may want to address. The places where, I think, it makes sense to address it somehow are:

- [ ] README.md
- [ ] 404.md
- [ ] CODE_OF_CONDUCT.md
- [ ] CONTRIBUTING.md
- [ ] foreword.md
- [ ] print-cover.html
- [ ] publiccode.yml

And, of course, GOVERNANCE.md, but there is already a [suggestion for that change](https://github.com/standard-for-public-code/standard-for-public-code/pull/1023).

Finally, [the theme](https://github.com/publiccodenet/jekyll-theme) this repository is using for the Jekyll build of the site, is a branded one too. How to address that might be worth its own issue, as it is a larger change, possibly addressed by forking the theme.

**Comment by janderssonse** [comment]
Will start looking at this tomorrow.

**Comment by ElenaFdR** [comment]
It looks like it would be helpful to have a short explanation of the relationship between this and the Foundation for Public Code, to help:

- people who know something about the Standard but may not know why/that this is now the leading version
- people who are complete newcomers, but need to know why to trust this (ie, it was initiated by a team of experts at a credible institution, and has since been moved to community governance/maintenance).

I propose something like this, based on [governance.md](/governance):

- The Standard for Public Code is a community governed project. It was started by the [Foundation for Public Code](https://publiccode.net/) in 2019.
- Started by the [Foundation for Public Code](https://publiccode.net/) in 2019, the Standard for Public Code has been [community governed since 2024](https://blog.publiccode.net/community%20call/2024/03/27/notes-from-community-call-26-march-2024.html).

**Comment by janderssonse** [comment]
@ElenaFdR Agree, and I like your suggestions. Currently it would be rather confusing for someone looking at this coming the outside and not knowing the history.

**Comment by jgroenen** [comment]
How to go about: "To raise a concern, please email [directors@publiccode.net](mailto:directors@publiccode.net)." (Code of Conduct)?

**Comment by ericherman** [comment]
> How to go about: "To raise a concern, please email [directors@publiccode.net](mailto:directors@publiccode.net)." (Code of Conduct)?

This is one to bring up on the next call -- what *is* a good way to raise a potential CoC issue?

What relatively confidential channels exist?

Would some steering team members be willing to list their (work?) email addresses?
If so, then maybe an invitation to email one of the steering team members with a published address could work?

The only semi-confidential channel I know of provided by the platform is for [privately-reporting-a-security-vulnerability](
https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability
), but maybe that mechanism would also make sense for raising a potential CoC issue?

**Comment by ixuz** [comment]
> > How to go about: "To raise a concern, please email [directors@publiccode.net](mailto:directors@publiccode.net)." (Code of Conduct)?
> 
> This is one to bring up on the next call -- what _is_ a good way to raise a potential CoC issue?
> 
> What relatively confidential channels exist?
> 
> Would some steering team members be willing to list their (work?) email addresses? If so, then maybe an invitation to email one of the steering team members with a published address could work?
> 
> The only semi-confidential channel I know of provided by the platform is for [privately-reporting-a-security-vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability), but maybe that mechanism would also make sense for raising a potential CoC issue?

There is a specific GitHub feature for managing this kind of issue/abuse, see [managing reported content in your organization](https://docs.github.com/en/communities/moderating-comments-and-conversations/managing-reported-content-in-your-organizations-repository).

> When disruptive behavior occurs in your community, GitHub offers tools for applying your code of conduct and de-escalating conflict. For example, you can lock a conversation to defuse a heated interaction. For more information, see Moderating comments and conversations.

This feature is activated on our organization/repository.

Let's discuss this further in the next community call.

**Comment by Ainali** [comment]
I think with the recent changes in the Code of conduct that we have addressed all parts of this issue. I'll close this now and if we missed something, that can be raised as separate issues.
